### PR TITLE
Change job link in humans.txt

### PR DESCRIPTION
**Pull request by bilbof**
Will change https://www.gov.uk/humans.txt

### CODE REVIEW

**Comment by bilbof** [comment]
The route is here: https://github.com/alphagov/static/blob/c8346267ecbf29ac9c8b1152ce2e8afaf1976e37/lib/tasks/publishing_api.rake#L8-L14

**Comment by chao-xian** [comment]
Where's the "Thanks Martha"?

**Comment by issyl0** [comment]
@chao-xian Also here: https://github.com/alphagov/static/blob/6a0d8a619e56063c81c193458bff92618f163306/app/views/root/homepage.html.erb#L13.

**Comment by bilbof** [comment]
@chao-xian suggest an edit! :-)

File: public/humans.txt [hunk 1]
```diff
@@ -1 +1 @@
-GOV.UK is built by a team at the Government Digital Service in London. If you'd like to join us, see https://gds.blog.gov.uk/jobs.
+GOV.UK is built by a team at the Government Digital Service in London. If you'd like to join us, see https://jobs.jobvite.com/gds.
```
Comment (chao-xian) [inline public/humans.txt:1]:
```suggestion
GOV.UK is built by a team at the Government Digital Service in London (with thanks to Martha!). If you'd like to join us, see https://jobs.jobvite.com/gds.
```

**Pull request merged**